Quantum-Control-Applications-QuAM/Superconducting/quam_experiments/analysis/fit.py

```diff
@@ -1,10 +1,9 @@
 from matplotlib import pyplot as plt
```

Comment:
Why not putting this file under `quam_libs`?

Reply:
Agreed with @deanpoulos, to me `quam_builder/quam_libs` would only contain things directly relevant to quam, so the components and the tools to build a quam.
A second distinction we could make is that `quam_builder` is versioned, whereas this is not

Reply:
@nulinspiratie Just to clarify the comment: the idea is to have two different repos:
1. `quam_builder` for the quam components and builder functions
2. `quam_libs` for utility functions used in the nodes (qua_dataset, save_utils, plotting_utils...)
I 100% agree that these shouldn't belong to `quam_builder`, but maybe` quam_libs` makes sense.

@deanpoulos exactly, if we don't include those into `quam_libs`, I am also questioning its utility. To me it is a way of ensuring proper testing and versioning of these general (not node-specific) functions which doesn't seem easy/possible with `quam_experiments`. Also these functions are also most likely to be used across different quam types and qubit platforms which justifies being part of a separate repo. What do you think?